R/pkg/R/install.R

```diff
@@ -54,7 +54,7 @@
 #'                 }
 #' @param overwrite If \code{TRUE}, download and overwrite the existing tar file in localDir
 #'                  and force re-install Spark (in case the local directory or file is corrupted)
-#' @return \code{install.spark} returns the local directory where Spark is found or installed
+#' @return the (invisible) local directory where Spark is found or installed
```

Comment:
it wasn't actually - it was always invisible - see [here](https://github.com/apache/spark/pull/16589/files/9b21c1125998b09607dc40c57dcd579d322b6f6e#diff-48b5d60aba76122785461e1c2b125f51R150), just not documented as such. Also referencing the method name and saying `returns` again is redundant as this would go to the return section of the generated doc.